src/components/VmDetails/cards/DetailsCard/NextRunChangeConfirmationModal.js

```diff
@@ -1,39 +1,21 @@
 import React, { useContext } from 'react'
 import PropsTypes from 'prop-types'
-import {
-  MessageDialog,
-  Button,
-  Icon,
-  noop,
-} from 'patternfly-react'
 import { MsgContext } from '_/intl'
+import ConfirmationModal from '_/components/VmActions/ConfirmationModal'
 
 const NextRunChangeConfirmationModal = ({ show, onCancel, onSave, onSaveAndRestart }) => {
   const { msg } = useContext(MsgContext)
   return (
-    <MessageDialog
+    <ConfirmationModal
       show={show}
-      onHide={onCancel}
+      onClose={onCancel}
       title={msg.nextRunConfirmTitle()}
-      icon={<Icon type='pf' name='warning-triangle-o' />}
 
-      primaryContent={<div className='lead'>{msg.nextRunConfirmContent()}</div>}
-      secondaryContent={<div>{msg.nextRunConfirmContentDetail()}</div>}
+      body={msg.nextRunConfirmContent()}
+      subContent={msg.nextRunConfirmContentDetail()}
```

Comment:
:thinking: Double check against PF4 guidelines

Title: "Some changes require a restart"
Content: "Some configurations changes will not be able to take effect until the Virtual Machine is restarted.  A power cycle needs to take place to pick up this new configuration."